Sources/ArcGISToolkit/Components/Popups/PopupMedia/Charts/ChartData.swift

```diff
@@ -28,9 +28,7 @@ struct ChartData: Identifiable {
     ///   - value: The value of the data.
     init(label: String, value: Any) {
         self.label = label
-        if let int32 = value as? Int32 {
```

Comment:
I'm not sure if this needs to be removed. Since there won't be `Int32` that gets exposed to the users, I think it might be OK to remove, but let me know if it has other usage!

Reply:
I think this change is okay

Reply:
@zkline101 The `value` in use here is coming from Core. Are all Core values converted to Int? Or can they still be `Int32`.

The value is essentially a numeric field in a table.

Reply:
> Are all Core values converted to Int? 

If the core value is exposed as an `Any` but the underlying type is an integer then yes that underlying type will be exposed to us as an `Int`. Does that help?

Reply:
@zkline101 Yes, that helps a lot, thanks.  @yo1995 It's OK to remove that check for `Int32`, as they all should be `Int`s.

Reply:
Thanks for the clarification.